src/modules/reading-room/services/visitor-space/vistor-space.service.ts

```diff
@@ -11,7 +12,9 @@ export class VistorSpaceService {
 	public static async getAll(
 		searchInput = '',
 		page = 0,
-		size = 20
+		size = 20,
+		orderProp: ReadingRoomOrderProps = ReadingRoomOrderProps.CreatedAt,
+		orderDirection: OrderDirection = OrderDirection.desc
```

Comment:
Als default waarde toevoegen aan query param config op relevante pagina's, niet in service.